### PR TITLE
Scope get event id endpoint to admins only

### PR DESCRIPTION
**Pull request by jeffplays2005**
* Updated endpoint

* Updated interfaces

* Updated tests

* Generate swagger

### CODE REVIEW

File: server/src/service-layer/controllers/AdminController.ts [hunk 1]
```diff
@@ -788,4 +791,25 @@ export class AdminController extends Controller {
     }
     this.setStatus(200)
   }
+
+  @Get("{id}")
```
Comment (bcho892) [inline server/src/service-layer/controllers/AdminController.ts:795]: wrong endpoint

File: server/src/middleware/tests/AdminController.test.ts [hunk 2]
```diff
@@ -850,4 +850,29 @@ describe("AdminController endpoint tests", () => {
...
+      physical_start_date: dateToFirestoreTimeStamp(new Date()),
+      location: "UASC",
+      sign_up_start_date: dateToFirestoreTimeStamp(new Date()),
+      sign_up_end_date: dateToFirestoreTimeStamp(new Date())
+    }
+    const eventService = new EventService()
+
+    it("should return the event details for a valid event ID", async () => {
+      const { id: id1 } = await eventService.createEvent(event1)
+      const res = await request.get(`/events/${id1}`).send()
```
Comment (bcho892) [inline server/src/middleware/tests/AdminController.test.ts:865]: wrong endpoint

**Review by bcho892 — CHANGES_REQUESTED** [review]

File: server/src/middleware/tests/AdminController.test.ts (hunk 2 above)
Comment (jeffplays2005) [reply server/src/middleware/tests/AdminController.test.ts:865]: Oops yep noticed that, will push a change now

**Comment by jeffplays2005** [comment]
I forgot to use a jwt token, ill make another push, please review after!

**Review by bcho892 — APPROVED** [review]
LGTM

**Pull request merged**